base/dsp.jl

```diff
@@ -121,9 +121,11 @@ function deconv{T}(b::StridedVector{T}, a::StridedVector{T})
 end
 
 """
-    conv(u,v)
+    conv(u, v)
 
-Convolution of two vectors. Uses FFT algorithm.
+Convolves two vectors. (If `u` and `v` describe the coefficients of two polynomials,
```

Comment:
Typically in describing functionality in docstrings, we stick to the imperative. So this should be "convolve" instead of "convolves."